samples/Samples.Schemas.Chat/Services/Chat.cs

```diff
@@ -20,7 +20,7 @@ public IEnumerable<Message> GetAllMessages()
     public IEnumerable<Message> GetMessageFromUser(string from)
     {
         lock (_messages)
-            return _messages.Where(x => string.Equals(x.From, from, StringComparison.InvariantCultureIgnoreCase)).ToList();
+            return _messages.Where(x => string.Equals(x.From, from, StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
I don't understand the logic of the change here.  Did it violate a rule?  Sorta puzzled...

Reply:
Yes, https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1309